docs/setup_config.rst

```diff
@@ -0,0 +1,40 @@
+Setup configuration
+===================
+
+Loading notification configuration from a YAML file
+***************************************************
+
+This library provides a ``ConfigurationStep``
+(from the library ``django-setup-configuration``, see the
+`documentation <https://github.com/maykinmedia/django-setup-configuration>`_
+for more information on how to run ``setup_configuration``)
+to configure the notification configuration.
+
+To add this step to your configuration steps, add ``django_setup_configuration`` to ``INSTALLED_APPS`` and add the following setting:
+
+    .. code:: python
+
+        SETUP_CONFIGURATION_STEPS = [
+            ...
+            "notifications_api_common.contrib.setup_configuration.steps.NotificationConfigurationStep"
+            ...
+        ]
+
+The YAML file that is passed to ``setup_configuration`` must set the
+``notifications_config_enable`` flag to ``true`` to enable the step. All fields under ``notifications_config`` are optional.
```

Comment:
Is there going to be a use-case where the `notifications_api_service_identifier` is not going to be defined? I'm unsure why an user would configure this without also defining a service. If I read it correctly the other options have no effect whenever that is the case.

Reply:
I initially thought that Open Notificaties didn't need to specify this, but it turns out that in order to be able to subscribe to the `autorisaties` channel, you do need to specify this.

I guess we could make it required to avoid confusion

Reply:
@swrichards would you agree making the `notifications_api_service_identifier` required (and thereby a `Service`) ?